src/components/layout/modal/browser_update_alert_modal.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react'
+import { ModalRefType } from 'components/elements/modal'
+import { Localize } from 'components/localization'
+import { Modal, Header } from 'components/elements'
+import { useUserBrowser } from 'components/hooks/use-user-browser'
+import { CookieStorage } from 'common/storage'
+
+const cookie_key = 'browser_update_alert_modal_shown'
```

Comment:
Please move this variables on constants.ts

Reply:
done